examples/connectivity/plot_adhd_covariance.py

```diff
@@ -8,11 +8,24 @@
  
 plotted_subject = 0  # subject to plot
 
+import numpy as np
 
 import matplotlib.pyplot as plt
+
+from nilearn import plotting, image
 from nilearn.plotting import cm
 
 
+def plot_connectome(cov, atlas_maps, **kwargs):
+    imgs = image.iter_img(msdl_atlas_dataset.maps)
+    regions_coords = np.array([
+        map(np.asscalar, plotting.find_xyz_cut_coords(img)) for img in imgs])
+    np.random.seed(42)
+    node_colors = np.random.rand(39, 3)
```

Comment:
I am wondering if it would not be interesting to use as node colors, simply the ordering of the node in the atlas. It would make the code simpler, and I believe that it has been chosen in a way that makes sense.